script/Ciel/Day 3/03_01_CIEL03_3_2B.txt

```diff
@@ -1,52 +1,59 @@
 [sha:e0e57782c96b49cc2668158577cd47a03759a0fa]{
 -- Page 10004, Offset 5164.
 -- ―――――――さむ、い。
-―――――――It's... cold.
+It's cold...
 }
 [sha:04cb3fc9bef22c18e826de16f21cbdea64cdbe5a]{
 -- Page 10004, Offset 5165.
 -- いつのまにか、日が落ちている。
- Before I know it, the sun has gone down.
+At some point, the sun started to set.
 }
 [sha:0c6a1d996726f1a70ada3b67af6ac54ecd9863a4]{
 -- Page 128, Offset 5166.
 -- ―――――――いまは、もうどれくらいだろう。
-――――――― I wonder how long it's been now.
+I wonder how long I've been here...
 }
 [sha:fd5d834112e354e3bc3b7568d2085cf5f21c635e]{
 -- Page 128, Offset 5167.
 -- よくわからない。
- I'm not really sure.
+I'm not really sure.
 }
 [sha:786830ae19399b8a40d1b171d3e68ae1ac56b5a5]{
 -- Page 128, Offset 5168.
 -- 鼓膜にはざあざあと、ノイズめいた音が響いている。
- My eardrums are full of rhythmic white noise.
+// Reading this ざあざあ as the rain that's falling on the BGM track rather than a low roaring white noise of shock
+A low droning hiss echoes in my ears.
 }
 [sha:94823b2e96f953fa9db962b1239083e01152450d]{
 -- Page 129, Offset 5169.
 -- ざあ。ざあ。ざあ。
- %{i}Pitt%{/i}. %{i}Patt%{/i}. %{i}Pitapat%{/i}.
+// Pitter patter feels too discrete, but there's no great en equivalent for the
+// ザーーー of steady curtains of rain
+%{i}Fshhhhhhhh%{/i}.
 }
 [sha:5440d4a6cf248f4a2888b5655eeb2f30994ccda3]{
 -- Page 129, Offset 5170.
 -- 音はやまず、ひどくさむい。
-The sound won't stop, and I'm terribly cold.
+The sound won't let up, and I'm terribly cold.
 }
 [sha:16a15a4e5585ed1df78f3abc576346d4d8a414fc]{
 -- Page 129, Offset 5171.
 -- このまま―――ベンチに座ったままでいたら、死んでしまうような気がする。
- I get the feeling I'll die if I just sit here on this bench.
+I get the feeling I'll die if I just sit here on this bench.
 }
 [sha:aa5c36c8c99f81c8dde2cbae3053cdfb1f70f5d7]{
 -- Page 129, Offset 5172.
 -- ……死んでしまえるような、気がする。
- ...The feeling I'll... die.
+The feeling I'll... die.
 }
 [sha:2dfe5b44fe9aa69639eb250cae5ad8a7dfa993c1]{
 -- Page 130, Offset 5173.
 -- ザザ鳴りの神経、ザラついた体温。
-The nerves picking up %{i}that sound%{/i}, my coarse body temperature.
+// Reading ザザ as a pins and needles/numbness deal
+// A _gritty_ body temperature?? Goosebumps? Or cooled to the point of 'coagulating',
```

Comment:
I mean... that's what it says. double check with petrikow, but this is _very_ liberally tl'd